.github/workflows/go.yml

```diff
@@ -0,0 +1,37 @@
+name: Go
```

Comment:
Why is this workflow here? I though there would be a single lint workflow for this PR.

Reply:
@stianst having separate flows allows you to distinguish between the failures generated by the build and failures generate by the lack of formatting for example. Whether or not they should be in a single flow, again, it's your choice.
Most of the projects out there prefer to use it in separate flows.